Source/VideoBlockViewController.swift

```diff
@@ -458,6 +459,16 @@ class VideoBlockViewController : OfflineSupportViewController, CourseBlockViewCo
     }
     
     //MARK: - VideoPlayerDelegate methods
+    
+    func playerDidFinishLoad(videoPlayer: VideoPlayer) {
+        if let frame = videoPlayer.controls?.getBottomBarPosition() {
+            if let contentView = parent as? CourseContentPageViewController {
+                contentView.restrictedPaginationAreaStart = frame.origin.y
+                contentView.restrictedPaginationAreaEnd = (frame.origin.y) + (frame.size.height)
+            }
+        }
+    }
```

Comment:
This can be formatted as

```suggestion
func playerDidFinishLoad(videoPlayer: VideoPlayer) {
        if let frame = videoPlayer.controls?.bottomBarFrame, let contentView = parent as? CourseContentPageViewController {
            contentView.restrictedPaginationAreaStart = frame.origin.y
            contentView.restrictedPaginationAreaEnd = frame.origin.y + frame.size.height
        }
    }
 
```

Reply:
done